lib/internal/fs/read_file_context.js

```diff
@@ -79,7 +79,8 @@ class ReadFileContext {
     } else {
       buffer = this.buffer;
       offset = this.pos;
-      length = Math.min(kReadFileBufferLength, this.size - this.pos);
+      // Read up to 256kb.
+      length = Math.min(256 * 1024, this.size - this.pos);
```

Comment:
If I understand this correctly, this should work fine, but may be we need to add a way for the user to define the min buffer size (if no options, fall back to `256kb`)? Will that make sense?

Reply:
I would rather not do this in this PR. I personally just want to make the default better. The only reason for a user to set this value manually would be for big files. If a user wants this extra performance boost they should be aware of the trade off with the thread pool and I doubt that this is the case for most developers. If someone indeed knows about this trade off, they can also load the file with other `fs` functions directly.